clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -136,7 +140,7 @@
  *           follower can subscribe back to VT using the recently updated VT offset.
  */
 public class LeaderFollowerStoreIngestionTask extends StoreIngestionTask {
-  private static final Logger LOGGER = LogManager.getLogger(LeaderFollowerStoreIngestionTask.class);
+  static Logger LOGGER = LogManager.getLogger(LeaderFollowerStoreIngestionTask.class);
```

Comment:
Can it remain private, even though it is not final anymore? Technically, we should make it in lower case since it's not a constant anymore, but I'm fine with omitting that in the interest of minimizing the diff size...